SECURITY.md

```diff
@@ -0,0 +1,2 @@
+## Security Best practices
+1. [https://snyk.io/blog/ten-npm-security-best-practices/](https://snyk.io/blog/ten-npm-security-best-practices/)
```

Comment:
Can you please expand on the list here for the quick reference?

Reply:
sure